docs/CONTRIBUTING.md

```diff
@@ -7,6 +7,24 @@ Contributing to NeoForge
 
 3) Follow the code style of the class you're working in (braces on newlines & spaces instead of tabs in Forge classes, inline brackets in patches, etc).
 
+## Workflow
+
+0. Have preliminary discussions on Discord (`#neoforge-github`)
+1. Fork the repository
+2. Check out your fork
+3. Make a branch
+4. Run `gradlew :neoforge:setup` from the project root
```

Comment:
I would run `gradlew setup` which also sets up the `base` project.

Reply:
That's what the above discussion is on :p